.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.15.5
+current_version = 1.16
```

Comment:
Can you remove the commits that change the version string? This is something that's done at release time.